src/Symfony/Component/PropertyInfo/Extractor/ReflectionExtractor.php

```diff
@@ -272,7 +291,7 @@ public function getReadInfo(string $class, string $property, array $context = []
     /**
      * {@inheritdoc}
      */
-    public function getWriteInfo(string $class, string $property, array $context = []): PropertyWriteInfo
+    public function getWriteInfo(string $class, string $property, array $context = []): ?PropertyWriteInfo
```

Comment:
This will go into another patch targeting 5.1 (some of the other changes in this class as well)